atmo/utils/provisioning.py

```diff
@@ -38,17 +38,20 @@ def cluster_start(user_email, identifier, size, public_key, emr_release):
     ]
 
     if num_instances > 1:
+        market = 'SPOT' if settings.AWS_CONFIG['USE_SPOT_INSTANCES'] == 'true' else 'ON_DEMAND'
```

Comment:
Use `'SPOT' if settings.AWS_CONFIG['USE_SPOT_INSTANCES'] else 'ON_DEMAND'` once having added the cast parameter above